packages/app/pages/index.tsx

```diff
@@ -0,0 +1,21 @@
+import type { NextPage } from "next";
+import Head from "next/head";
+import styles from "../styles/Home.module.css";
+
+const Home: NextPage = () => {
+  return (
+    <div className={styles.container}>
+      <Head>
+        <title>NFT Passports</title>
+        <meta name="description" content="App | NFT Passports" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <main className={styles.main}>
+        <h1>NFT Passports Dashboard</h1>
```

Comment:
took out most of the create-next-app index boilerplate and added `title` + `description` above